cocos/physics/framework/physics-interface.ts

```diff
@@ -263,15 +263,15 @@ export class CharacterControllerContact {
      * @zh
      * 碰撞中的角色控制器。
      */
-    selfCCT: any = null;
+    selfController: CharacterController = undefined!;
```

Comment:
Why use undefined

Reply:
Should we avoid using `any`?

Reply:
Because he might want to reuse this class?